momepy/functional/_dimension.py

```diff
@@ -157,3 +169,162 @@ def perimeter_wall(
     ].values
 
     return results
+
+
+def street_profile(
+    left: GeoDataFrame,
+    right: GeoDataFrame,
```

Comment:
```suggestion
    streets: GeoDataFrame,
    buildings: GeoDataFrame,
```

Probably better to be explicit.